src/Components/CreateImageWizard/steps/Review/ReviewStep.tsx

```diff
@@ -244,6 +252,23 @@ const Review = ({ snapshottingEnabled }: { snapshottingEnabled: boolean }) => {
           <OscapList />
         </ExpandableSection>
       )}
+      {compliancePolicy && complianceType === 'compliance' && (
+        <ExpandableSection
+          toggleContent={composeExpandable(
+            'Compliance',
+            'revisit-compliance',
+            'step-oscap'
```

Comment:
```suggestion
            'step-compliance'
```

Reply:
It's the same step now, so I think `step-oscap` is correct.

Reply:
Right, sorry, was still thinking in separate steps :bow: